src/components/app/Home.js

```diff
@@ -339,16 +339,12 @@ class Home extends React.PureComponent<HomeProps, HomeState> {
           className="homeInstructions"
           data-testid="home-enable-popup-instructions"
         >
-          <div className="homeInstructionsLeft">
-            <div style={{ textAlign: 'center' }}>
-              <img
-                className="homeSectionScreenshot"
-                src={PerfScreenshot}
-                alt="screenshot of profiler.firefox.com"
-              />
-            </div>
-          </div>
-          <div className="homeInstructionsRight">
+          <img
```

Comment:
Could you please add comments about where the grids start here? I find grid implementations to be less obvious from the markup on what's going on, and I think it would help me understand this in the future.

e.g.

`{/* Grid Column 1 */}`